Gems/Atom/Asset/Shader/Code/Source/Editor/ShaderAssetBuilder.cpp

```diff
@@ -639,6 +639,18 @@ namespace AZ
                             }
                         }
 
+#if !defined(_RELEASE)
```

Comment:
Please use #if defined(AZ_ENABLE_TRACING) instead. That directly corresponds to AZ_Warning, AZ_Error, and AZ_Assert being present.